var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -617,6 +633,10 @@ def setup_build_environment(self, env):
             env.set('INCLUDEDIR', spec['protobuf'].prefix.include)
 
     def patch(self):
+        filter_file(
+            '"-U_FORTIFY_SOURCE",',
+            "\"-U_FORTIFY_SOURCE\", \"-I%s\"," % self.spec['protobuf'].prefix.include,
```

Comment:
```suggestion
            '"-U_FORTIFY_SOURCE", "-I%s",' % self.spec['protobuf'].prefix.include,
```

Reply:
i have taken the changes suggested by you.